tools/github_webhook_processor.php

```diff
@@ -197,45 +295,83 @@ function checkchangelog($payload, $merge = false) {
 			case 'wip':
-				$currentchangelogblock[] = array('type' => 'wip', 'body' => $item);
+				if($item != 'added a few works in progress')
+					$currentchangelogblock[] = array('type' => 'wip', 'body' => $item);
 				break;
 			case 'rsctweak':
 			case 'tweaks':
 			case 'tweak':
-				$currentchangelogblock[] = array('type' => 'tweak', 'body' => $item);
+				if($item != 'tweaked a few things') {
+					$tags[] = "Tweak";
+					$currentchangelogblock[] = array('type' => 'tweak', 'body' => $item);
+				}
 				break;
 			case 'soundadd':
-				$currentchangelogblock[] = array('type' => 'soundadd', 'body' => $item);
+				if($item != 'added a new sound thingy') {
+					$tags[] = "Sound";
+					$currentchangelogblock[] = array('type' => 'soundadd', 'body' => $item);
+				}
 				break;
 			case 'sounddel':
-				$currentchangelogblock[] = array('type' => 'sounddel', 'body' => $item);
+				if($item != 'removed an old sound thingy') {
+					$tags[] = "Sound";
+					$tags[] = "Revert/Removal";
+					$currentchangelogblock[] = array('type' => 'sounddel', 'body' => $item);
+				}
 				break;
 			case 'add':
 			case 'adds':
 			case 'rscadd':
-				$currentchangelogblock[] = array('type' => 'rscadd', 'body' => $item);
+				if($item != 'Added new things' && $item != 'Added more things') {
+					$tags[] = "Feature";
+					$currentchangelogblock[] = array('type' => 'rscadd', 'body' => $item);
+				}
 				break;
 			case 'del':
 			case 'dels':
 			case 'rscdel':
-				$currentchangelogblock[] = array('type' => 'rscdel', 'body' => $item);
+				if($item != 'Removed old things') {
+					$tags[] = "Revert/Removal";
+					$currentchangelogblock[] = array('type' => 'rscdel', 'body' => $item);
+				}
 				break;
 			case 'imageadd':
-				$currentchangelogblock[] = array('type' => 'imageadd', 'body' => $item);
+				if($item != 'added some icons and images') {
+					$tags[] = "Sprites";
+					$currentchangelogblock[] = array('type' => 'imageadd', 'body' => $item);
+				}
 				break;
 			case 'imagedel':
-				$currentchangelogblock[] = array('type' => 'imagedel', 'body' => $item);
+				if($item != 'deleted some icons and images') {
+					$tags[] = "Sprites";
+					$tags[] = "Revert/Removal";
+					$currentchangelogblock[] = array('type' => 'imagedel', 'body' => $item);
+				}
 				break;
 			case 'typo':
 			case 'spellcheck':
-				$currentchangelogblock[] = array('type' => 'spellcheck', 'body' => $item);
+				if($item != 'fixed a few typos') {
+					$tags[] = "Grammar and Formatting";
+					$currentchangelogblock[] = array('type' => 'spellcheck', 'body' => $item);
+				}
 				break;
 			case 'experimental':
 			case 'experiment':
-				$currentchangelogblock[] = array('type' => 'experiment', 'body' => $item);
+				if($item != 'added an experimental thingy')
+					$currentchangelogblock[] = array('type' => 'experiment', 'body' => $item);
+				break;
+			case 'balance':
+			case 'rebalance':
+				if($item != 'rebalanced something'){
+					$tags[] = "Balance/Rebalance";
```

Comment:
Minor as fuck nitpick You use `"` when you don't do substitutions, causing needless overhead. Theses should be single quotes for the tag assignments